File: CakeKnife/app/src/main/java/cc/lixiaoyu/cakeknife/sample/MainActivity.java
package cc.lixiaoyu.cakeknife.sample;

import androidx.appcompat.app.AppCompatActivity;
import cc.lixiaoyu.cakeknife.CakeKnife;
import cc.lixiaoyu.cakeknife.annotations.BindView;

import android.os.Bundle;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @BindView(R.id.main_textview)
    public TextView mTextView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        CakeKnife.bind(this);
        mTextView.setText("Owen");
    }
}

File: JetpackPractice/app/src/main/java/life/lixiaoyu/jetpackpractice/main/MainActivity.kt
package life.lixiaoyu.jetpackpractice.main

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView
import life.lixiaoyu.jetpackpractice.R

class MainActivity : AppCompatActivity() {

    private var recyclerview: RecyclerView? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        recyclerview = findViewById(R.id.recyclerview)
        recyclerview?.let {
            it.adapter = MainAdapter()
            it.layoutManager = LinearLayoutManager(this)
        }
    }
}